typescript/rest-nestjs/package.json

```diff
@@ -21,9 +21,9 @@
     "test:e2e": "jest --config ./test/jest-e2e.json"
   },
   "dependencies": {
-    "@nestjs/common": "7.6.9",
-    "@nestjs/core": "7.6.9",
-    "@nestjs/platform-express": "7.6.9",
+    "@nestjs/common": "7.6.8",
+    "@nestjs/core": "7.6.8",
+    "@nestjs/platform-express": "7.6.8",
```

Comment:
This will loose the version upgrade that seems to have happened in the meantime, but will probably fixed via renovate again later if you prefer not to have to undo this manually in the PR.